File: app/src/main/java/net/a8pade8/passwordsaver/data/PasswordSaverContract.java
package net.a8pade8.passwordsaver.data;

import android.provider.BaseColumns;

/**
 * Created by stanislav on 07.12.16.
 */

public final class PasswordSaverContract {

    private PasswordSaverContract() {
    }

    public static final String DATA_BASE = "psdb.db";

    public static final class Passwords implements BaseColumns {
        final static String TABLE_PASSWORDS = "passwords";

        final static String _ID = BaseColumns._ID;
        public final static String COLUMN_RESOURCE = "resource";
        public final static String COLUMN_LOGIN = "login";
        public final static String COLUMN_PASSWORD = "password";
        public final static String COLUMN_COMMENT = "comment";
    }
}

File: app/src/main/java/net/a8pade8/passwordsaver/data/PasswordSaverContract.kt
package net.a8pade8.passwordsaver.data

import android.provider.BaseColumns

object PasswordSaverContract {

    const val DATA_BASE = "psdb.db"

    object Passwords : BaseColumns {
        const val TABLE_PASSWORDS = "passwords"
        const val _ID = BaseColumns._ID
        const val COLUMN_RESOURCE = "resource"
        const val COLUMN_LOGIN = "login"
        const val COLUMN_PASSWORD = "password"
        const val COLUMN_COMMENT = "comment"
        const val COLUMN_FAVORITE = "favorite"
    }
}